plugin.xml

```diff
@@ -35,9 +35,9 @@
   </engines>
 
   <!-- Specific versions of dependencies required by the Mobile SDK plugin -->
-  <dependency id="cordova-plugin-whitelist" url="https://github.com/apache/cordova-plugin-whitelist" commit="1.2.0" />
-  <dependency id="cordova-plugin-device" url="https://github.com/apache/cordova-plugin-device" commit="r1.0.1" />
-  <dependency id="phonegap-plugin-push" url="https://github.com/phonegap/phonegap-plugin-push" commit="1.5.0" />
+  <dependency id="cordova-plugin-whitelist" version="1.2.0" />
```

Comment:
`Cordova` appears to have changed the way they handle the `commit` argument. They completely ignore it if it's not a `SHA-1` (this used to work). As a result, the latest version of the `PushPlugin` was being pulled in, which brought `Firebase` with it and a slew of other problems (needs `Google Services`), etc. They did add a new parameter called `version` that seems to work though. Hybrid apps are working now. @wmathurin